### DISCUSSION TITLE
how do I disable plugins from cli?

### DISCUSSION THREAD

**Discussion by ralyodio** [btcpayserver/btcpayserver#5560] [Technical Support] [answered]
I installed podcaster and now I just get an empty page on every page.

HOw can I disable that plugin from the cli?

**Comment by dennisreimann** [comment, marked as answer]
If you are using the Docker deployment, you plugins should be located in `/var/lib/docker/volumes/generated_btcpay_pluginsdir/_data/`. You can go there and delete PodServer like this:

```
cd /var/lib/docker/volumes/generated_btcpay_pluginsdir/_data
rm -rf BTCPayServer.Plugins.PodServer
```

**Comment by ralyodio** [comment]
Thanks!
